ubuntu-advantage.1

```diff
@@ -101,10 +101,16 @@ package which has the common criteria artifacts. The artifacts include a
 configure script, a tarball with additional packages and post install scripts
 and an evaluated configuration guide. The artifacts will be installed in
 /usr/lib/common-criteria directory.
+.SH CIS (Canonical CIS Audit tooling)
+Enable CIS Auditing PPA and install CIS audit tool package
 .TP
 .B
-disable-cc-provisioning
-Disabling common criteria provisioning is not supported
```

Comment:
Did you remove disable-cc-provisioning from the manpage on purpose?

Reply:
That was an unintentional. I am thinking to add support to disable-cc-provisioning flag along with disable-cisaudit flag since they are very similar. Can I do it in this same PR?

Reply:
I think disable-cisaudit can be done here, but disable-cc-provisioning needs its own branch.

Reply:
OK